cheatsheets/Logging_Cheat_Sheet.md

```diff
@@ -10,27 +10,31 @@ Application logging should be consistent within the application, consistent acro
 
 ## Purpose
 
-Application logging should always be included for security events. Application logs are invaluable data for:
+Application logging should always be included for security events. Application logs are invaluable data for both security and operational use cases.
 
-- Identifying security incidents
-- Monitoring policy violations
+### Operational use cases
+
+- General debugging
 - Establishing baselines
-- Assisting non-repudiation controls (note that the trait non-repudiation is hard to achieve for logs because their trustworthiness is often just based on the logging party being audited properly while mechanisms like digital signatures are hard to utilize here)
+- Business process monitoring e.g. sales process abandonment, transactions, connections
 - Providing information about problems and unusual conditions
-- Contributing additional application-specific data for incident investigation which is lacking in other log sources
-- Helping defend against vulnerability identification and exploitation through attack detection
+- Performance monitoring e.g. data load time, page timeouts
+- Other business-specific requirements
+
+### Security use cases
 
 Application logging might also be used to record other types of events too such as:
 
-- Security events
-- Business process monitoring e.g. sales process abandonment, transactions, connections
 - Anti-automation monitoring
+- Identifying security incidents
+- Monitoring policy violations
```

Comment:
After 25+ years of me working in AppSec with CIRT and/or DFIR teams, I've observed that the CIRT / DFIR teams tend to split the log messages for security use cases into 2 main camps. One are log messages that they will use to trigger alerts (often combined with thresholds), and the other type are log events that is used as forensic evidence which they generally ignore until they open an investigation into a potential breach.

This division referenced here seems to be alluding to that. I'm not saying that we ought to provide more detail here, but was just curious if anyone else had had ever made similar observations.

Reply:
I haven't seen this, but I think from an advice perspective something like "Ask yourself 'is this a message someone will use to trigger an action, or as part of a larger investigation'?"